### PR TITLE
fix(shopify): fix back button redirect when error

### PR DESCRIPTION
**Pull request by cbernat**
Added redirect to control panel for back button when there is an error.

http://cdn.fromdoppler.com/doppler-webapp/build1147/#/integrations/shopify

### CODE REVIEW

**Comment by andresmoschini** [comment]
👍

File: src/components/Integrations/Shopify/Shopify.js [hunk 1]
```diff
@@ -118,9 +118,12 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
...
               </div>
               <footer className="dp-integration__actions">
-                <button type="button" className="dp-button button-medium primary-grey">
+                <a
+                  href={_('common.control_panel_url')}
+                  className="dp-button button-medium primary-grey"
+                >
                   {_('common.back')}
-                </button>
+                </a>
```
Comment (andresmoschini) [inline src/components/Integrations/Shopify/Shopify.js]: Is it not possible to avoid coding duplication for this button?

Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: I can add a fragment for this, yes 😄

Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: Fixed

**Pull request merged**